crm-project/src/main/java/CRM/utils/Validations.java

```diff
@@ -104,9 +104,9 @@ public static <T> T doesIdExists(Long id, JpaRepository repo) throws AccountNotF
         Optional<T> element = repo.findById(id);
         if (!element.isPresent()) {
             if (repo.getClass().getSimpleName().split("\\$")[0].equals(UserRepository.class.getSimpleName()))
-                throw new AccountNotFoundException(ExceptionMessage.NO_SUCH_ID.toString());
+                throw new AccountNotFoundException(ExceptionMessage.ACCOUNT_DOES_NOT_EXISTS.toString());
 
-            throw new NoSuchElementException(ExceptionMessage.NO_SUCH_ID.toString());
```

Comment:
Suggestion:
keep line 109 with NO_SUCH_ID
the upper one in 107 is ACCOUNT_DOES_NOT_EXSITS because its for userRepo only

Reply:
Fixed it
@shailevi23